data-prepper-plugins/kinesis-source/src/main/java/org/opensearch/dataprepper/plugins/kinesis/source/ClientFactory.java

```diff
@@ -0,0 +1,50 @@
+package org.opensearch.dataprepper.plugins.kinesis.source;
```

Comment:
I know we aren't following it consistently but I think it is better to name this as KinesisClientFactory.java. There are too many "ClientFactory.java" and it becomes search more difficult.